public/v2/apps/discourse.json

```diff
@@ -0,0 +1,177 @@
+                "environment": {
+                    "POSTGRESQL_HOST": "srv-captain--$$cap_appname-postgresql",
+                    "POSTGRESQL_ROOT_USER": "postgres",
+                    "POSTGRESQL_CLIENT_CREATE_DATABASE_NAME": "$$cap_db_name",
+                    "POSTGRESQL_CLIENT_CREATE_DATABASE_USERNAME": "$$cap_db_user",
+                    "POSTGRESQL_CLIENT_CREATE_DATABASE_PASSWORD": "$$cap_db_pass",
+                    "DISCOURSE_USERNAME": "testusername1234",
+                    "DISCOURSE_PASSWORD": "testpassword1234",
+                    "DISCOURSE_EMAIL": "testemail@gmail.com",
+                    "DISCOURSE_SITENAME": "$$cap_sitename",
+                    "DISCOURSE_POSTGRESQL_NAME": "$$cap_db_name",
+                    "DISCOURSE_POSTGRESQL_USERNAME": "$$cap_db_user",
+                    "DISCOURSE_POSTGRESQL_PASSWORD": "$$cap_db_pass",
+                    "DISCOURSE_HOSTNAME": "$$cap_appname.$$cap_root_domain",
+                    "REDIS_HOST": "srv-captain--$$cap_appname-redis",
+                    "SMTP_HOST": "$$cap_smtp_host",
+                    "SMTP_PORT": "$$cap_smtp_port",
+                    "SMTP_USER": "$$cap_smtp_user",
+                    "SMTP_PASSWORD": "$$cap_smtp_pass"
+                }
+            },
+            "$$cap_appname-sidekiq": {
+                "documentaion": "Taken from https://hub.docker.com/r/bitnami/discourse",
+                "notExposeAsWebApp": "true",
+                "depends_on": ["$$cap_appname"],
+                "volumes": ["$$cap_appname-data:/bitnami"],
+                "dockerfileLines": [
+                    "FROM docker.io/bitnami/discourse:$$cap_discourse_version",
+                    "CMD [\"nami\" , \"start\" , \"--foreground\" , \"discourse-sidekiq\"]"
+                ],
+                "environment": {
+                    "DISCOURSE_POSTGRESQL_NAME": "$$cap_db_name",
+                    "DISCOURSE_POSTGRESQL_USERNAME": "$$cap_db_user",
+                    "DISCOURSE_POSTGRESQL_PASSWORD": "$$cap_db_pass",
+                    "DISCOURSE_HOST": "srv-captain--$$cap_appname",
+                    "DISCOURSE_PORT": "3000",
+                    "DISCOURSE_HOSTNAME": "$$cap_appname.$$cap_root_domain",
+                    "POSTGRESQL_HOST": "srv-captain--$$cap_appname-postgresql",
+                    "REDIS_HOST": "srv-captain--$$cap_appname-redis",
+                    "SMTP_HOST": "$$cap_smtp_host",
+                    "SMTP_PORT": "$$cap_smtp_port",
+                    "SMTP_USER": "$$cap_smtp_user",
+                    "SMTP_PASSWORD": "$$cap_smtp_pass"
+                }
+            }
+        }
+    },
+    "instructions": {
+        "start": "Discourse is the 100% open source discussion platform built for the next decade of the Internet. Use it as a mailing list, discussion forum, long-form chat room, and more!",
+        "end": "Discourse is deployed and will be available in the next few minutes as $$cap_appname"
```

Comment:
Change this to something that makes it more clear that users have to wait:
```
Discourse image is deploy and the installation process has begun. The installation process takes about 5 minutes.
```

Reply:
Okay.